fsm/README.md

```diff
@@ -1,71 +1,29 @@
 # fsm
 
-Faecal sludge management (FSM) is a system that enables citizen to raise a request for septic tank cleaning with there ULB’s directly or reaching out to ULB counter. Citizen can track the application, make a payment for the charges and rate the service. This document contains the details about how to setup the fsm and describes the functionalities it provides.   
```

Comment:
why this content is removed
This PR is for adding UML daigram only

I think you did not take pull properly before pushing code.

Please check this and raised the proper PR